cluster-autoscaler/cloudprovider/clusterapi/clusterapi_converters.go

```diff
@@ -42,6 +42,22 @@ func newMachineDeploymentFromUnstructured(u *unstructured.Unstructured) *Machine
 		Status: MachineDeploymentStatus{},
 	}
 
+	machineSpec, found, err := unstructured.NestedMap(u.Object, "spec", "template", "spec")
+	if err == nil && found {
+		machineSpecUnstructured := unstructured.Unstructured{Object: machineSpec}
+		machineDeployment.Spec.Template.Spec.Labels = machineSpecUnstructured.GetLabels()
+		taints, _, _ := unstructured.NestedSlice(machineSpec, "Taints")
```

Comment:
When using the unstructured, everything is the way it would be written in `json`, so I think this should be lower cased
```suggestion
		taints, _, _ := unstructured.NestedSlice(machineSpec, "taints")
```

Reply:
yeah, i'll go back and clean this up. i wanted to share my early work to see if my approach was proper.